src/views/admin/PendingAccounts.vue

```diff
@@ -3,19 +3,25 @@
     <div class="container mt-5">
       <div class="columns is-tablet">
         <div class="column">
+           <loading v-model:active="this.isLoading"
+                 :can-cancel="false"
+                 :is-full-page="this.fullPage"/>
           <div
-            class="accounts card custom-card custom-card-padding"
+            v-if="!this.isLoading" class="accounts card custom-card custom-card-padding"
           >
-            <h2 class="custom-card-title">Comptes en attente de validation</h2>
-            <table class="table is-striped is-fullwidth">
-              <thead>
+         <div v-if="this.isEmptyList">Aucun compte en attente de validation</div>
```

Comment:
ou do not need to put `this.` to access local components properies in the template. `isEmptyList` would have been sufficient, just as `isLoading` and `fullPage` is enough.
Also, your `this.isEmptyList` switch is useless, because you know that your list is empty by `pendingUserAccounts.length === 0`, please remove that property from the component.

And then you can rewrite the line:
```
<div v-if="pendingUserAccounts.length === 0">Aucun compte en attente de validation</div>
```